libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -598,7 +598,7 @@ bool ContractCompiler::visit(FunctionDefinition const& _function)
 	}
 
 	for (ASTPointer<VariableDeclaration> const& variable: _function.returnParameters())
-		appendStackVariableInitialisation(*variable);
+		appendStackVariableInitialisation(*variable, true);
```

Comment:
What do you think about this style?
```suggestion
		appendStackVariableInitialisation(*variable, /* _provideDefaultValue = */ true);
```
The bare bool value is bugging me a bit here but a new enum just for this might be an overkill :)